enhancements/console/catalog-categories.md

```diff
@@ -0,0 +1,153 @@
+---
+title: catalog-categories
+authors:
+  - "@jerolimov"
+reviewers:
+  - TBD
+  - "@spadgett"
+  - "@cvogt"
+approvers:
+  - TBD
+creation-date: 2020-10-22
+last-updated: 2020-10-22
+status: implementable
+---
+
+# Customize Developer Catalog Categories
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [x] Design details are appropriately documented from clear requirements
+- [x] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+The OpenShift Console provides a "Developer Catalog" which enables the user to create apps and services based on Operator backed services, helm charts, templates, etc.
+
+Cluster admins can provide additional catalog items by installing operators or adding helm repositories. But the filter categories (All Items, Languages, Databases, Middleware, ...) are hard coded. This enhancement enable cluster admin to customize these category filters.
+
+## Motivation
+
+Customers who provide their own catalog items would like to customize the category filters such that they better represent their catalog items and make it easier for developers to navigate the catalog.
+
+### Goals
+
+- Extract the Developer Catalog categories into a CRD to allow cluster admins to customize them.
```

Comment:
This makes it sound like we're creating a new CRD instead of just updating the console config. I'd probably phrase it just to say we're adding a configuration option for customizing the categories.

Reply:
Updated this to

- Add configuration options to allow cluster admins to customize catalog categories.
- Provide a list of default values to the admin